src/core/mcir/spec.go

```diff
@@ -828,183 +828,183 @@ func FilterSpecsByRange(nsId string, filter FilterSpecsByRangeRequest) ([]TbSpec
-		sqlQuery = sqlQuery.And("Ebs_bw_Mbps <= ?", filter.Ebs_bw_Mbps.Max)
+		sqlQuery = sqlQuery.And("Ebs_bw_Mbps <= ?", filter.EbsBwMbps.Max)
 	}
 
-	if filter.Gpu_model != "" {
+	if filter.GpuModel != "" {
 		//sqlQuery += " AND `gpu_model` LIKE '%" + filter.Gpu_model + "%'"
-		filter.Gpu_model = RefineSpecName(filter.Gpu_model)
-		sqlQuery = sqlQuery.And("Gpu_model LIKE ?", "%"+filter.Gpu_model+"%")
+		filter.GpuModel = RefineSpecName(filter.GpuModel)
+		sqlQuery = sqlQuery.And("Gpu_model LIKE ?", "%"+filter.GpuModel+"%")
 	}
 
-	if filter.Num_gpu.Min > 0 {
+	if filter.NumGpu.Min > 0 {
 		//sqlQuery += " AND `num_gpu`>=" + fmt.Sprintf("%.6f", filter.Num_gpu.Min)
-		sqlQuery = sqlQuery.And("Num_gpu >= ?", filter.Num_gpu.Min)
+		sqlQuery = sqlQuery.And("Num_gpu >= ?", filter.NumGpu.Min)
 	}
-	if filter.Num_gpu.Max > 0 {
+	if filter.NumGpu.Max > 0 {
 		//sqlQuery += " AND `num_gpu`<=" + fmt.Sprintf("%.6f", filter.Num_gpu.Max)
-		sqlQuery = sqlQuery.And("Num_gpu <= ?", filter.Num_gpu.Max)
+		sqlQuery = sqlQuery.And("Num_gpu <= ?", filter.NumGpu.Max)
 	}
 
-	if filter.Gpumem_GiB.Min > 0 {
+	if filter.GpuMemGiB.Min > 0 {
 		//sqlQuery += " AND `gpumem_GiB`>=" + fmt.Sprintf("%.6f", filter.Gpumem_GiB.Min)
-		sqlQuery = sqlQuery.And("Gpumem_GiB >= ?", filter.Gpumem_GiB.Min)
+		sqlQuery = sqlQuery.And("Gpumem_GiB >= ?", filter.GpuMemGiB.Min)
 	}
-	if filter.Gpumem_GiB.Max > 0 {
+	if filter.GpuMemGiB.Max > 0 {
 		//sqlQuery += " AND `gpumem_GiB`<=" + fmt.Sprintf("%.6f", filter.Gpumem_GiB.Max)
-		sqlQuery = sqlQuery.And("Gpumem_GiB <= ?", filter.Gpumem_GiB.Max)
+		sqlQuery = sqlQuery.And("Gpumem_GiB <= ?", filter.GpuMemGiB.Max)
 	}
 
-	if filter.Gpu_p2p != "" {
+	if filter.GpuP2p != "" {
 		//sqlQuery += " AND `gpu_p2p` LIKE '%" + filter.Gpu_p2p + "%'"
-		filter.Gpu_p2p = RefineSpecName(filter.Gpu_p2p)
-		sqlQuery = sqlQuery.And("Gpu_p2p LIKE ?", "%"+filter.Gpu_p2p+"%")
+		filter.GpuP2p = RefineSpecName(filter.GpuP2p)
+		sqlQuery = sqlQuery.And("Gpu_p2p LIKE ?", "%"+filter.GpuP2p+"%")
 	}
 	if filter.EvaluationStatus != "" {
 		//sqlQuery += " AND `evaluationStatus` LIKE '%" + filter.EvaluationStatus + "%'"
 		filter.EvaluationStatus = RefineSpecName(filter.EvaluationStatus)
 		sqlQuery = sqlQuery.And("EvaluationStatus LIKE ?", "%"+filter.EvaluationStatus+"%")
 	}
 
-	if filter.EvaluationScore_01.Min > 0 {
+	if filter.EvaluationScore01.Min > 0 {
 		//sqlQuery += " AND `evaluationScore_01`>=" + fmt.Sprintf("%.6f", filter.EvaluationScore_01.Min)
-		sqlQuery = sqlQuery.And("EvaluationScore_01 >= ?", filter.EvaluationScore_01.Min)
+		sqlQuery = sqlQuery.And("EvaluationScore_01 >= ?", filter.EvaluationScore01.Min)
 	}
-	if filter.EvaluationScore_01.Max > 0 {
+	if filter.EvaluationScore01.Max > 0 {
 		//sqlQuery += " AND `evaluationScore_01`<=" + fmt.Sprintf("%.6f", filter.EvaluationScore_01.Max)
-		sqlQuery = sqlQuery.And("EvaluationScore_01 <= ?", filter.EvaluationScore_01.Max)
+		sqlQuery = sqlQuery.And("EvaluationScore_01 <= ?", filter.EvaluationScore01.Max)
```

Comment:
```suggestion
		sqlQuery = sqlQuery.And("EvaluationScore01 <= ?", filter.EvaluationScore01.Max)
```
SQL 쿼리에 대한 String이라 반영이 되지 않은 모양이네요..^^

이하 유사 구문에서도 해당 사항 반영이 필요해 보입니다.
EvaluationScore_0X -> EvaluationScore0X